packages/commons/src/events/state/flags.ts

```diff
@@ -24,6 +24,9 @@ function isPendingCertification(actions: Action[]) {
     if (type === ActionType.PRINT_CERTIFICATE) {
       return false
     }
+    if (type === ActionType.REQUEST_CORRECTION) {
+      return false
+    }
```

Comment:
Do we want `pending-certification` flag to be removed when someone makes a correction request? I'm thinking do we need to change the workqueue queries instead?